Nodejs/Product/Nodejs/Nodejs.csproj

```diff
@@ -910,6 +910,7 @@
       <IncludeInVSIX>true</IncludeInVSIX>
       <VSIXSubPath>ItemTemplates</VSIXSubPath>
     </Content>
+    <ZipProject Include="ProjectTemplates\Express4App\typings.json" />
```

Comment:
nit: I know VS chose this location, but can we just move this closer to the rest of the Express4App project template?